modules/installation-osp-local-disk-deployment.adoc

```diff
@@ -0,0 +1,296 @@
+// Module included in the following assemblies:
+//
+// * installing_openstack/deploying-openstack-with-rootVolume-etcd-on-local-disk.adoc
+
+:_mod-docs-content-type: Procedure
+[id="installation-osp-local-disk-deployment_{context}"]
+= Deploying {rh-openstack} on local disk
+
+.Prerequisites
+
+* You have an OpenStack cloud with a working Cinder, and it has at least 75 GB of available storage to accommodate 3 root volumes for the OpenShift control plane.
+
+* The OpenStack cloud must be deployed with Nova ephemeral storage that uses a local storage backend and not RBD.
+
+
+.Procedure
+
+[WARNING]
+====
+This procedure can testing etcd on local disk and should not be used on production cluster because in certain cases complete loss of the control plane can occur. For more information, see "Overview of backup and restore operation" under "Backup and restore".
```

Comment:
```suggestion
This procedure can test etcd on the local disk and should not be used on a production cluster, as in certain cases a complete loss of the control plane can occur. For more information, see the _Backup and restore guide: Overview of the backup and restore operation_.
```

Reply:
I've never seen italics used like that. I _think_ that is only for external references and not for internal references. https://www.ibm.com/docs/en/ibm-style?topic=references-citations#:~:text=Follow%20these%20general%20guidelines%20for%20citing%20one%20publication%20(such%20as%20a%20book%2C%20article%2C%20or%20web%20page)%20in%20another%20publication%3A